src/dialogs/admin/RejectDialog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <q-dialog v-model="rejectDialog" persistent>
+    <q-card
+      class="q-pa-md"
+      style="border-radius: 15px; background-color: black"
+    >
+      <q-card-section class="text-center">
+        <div class="text-h6 text-white">
```

Comment:
to conform to white background
```suggestion
        <div class="text-h6">
```